jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/list/ProcessInstanceListPresenter.java

```diff
@@ -160,6 +161,13 @@ public void callback(DataSet dataSet) {
         };
     }
 
+    protected boolean filterInitiator(String variableName, String variableValue, String initiator) {
+        if (variableName.equals("initiator") && variableValue.equals(initiator)) {
```

Comment:
perhaps a short version could be _return variableName.equals("initiator") && variableValue.equals(initiator) == false_

Reply:
yes you are right .